tox.ini

```diff
@@ -9,16 +9,25 @@ isolated_build = true
 
 [testenv]
 changedir = .tmp/{envname}
+setenv =
+    # Until a released numpy version is 3.12 compatible, we install the dev wheel
+    py312: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
```

Comment:
You can also use the 1.26 RC, too, it's on the normal pypi already.

Reply:
Not an issue anymore. I cleaned this up in a follow-up commit.